src/Bridges/Bridges.jl

```diff
@@ -328,7 +329,19 @@ function runtests(
                     Test.@test MOI.get(model, attr, ci) === nothing
                     start = _fake_start(constraint_start, set)
                     MOI.set(model, attr, ci, start)
-                    Test.@test MOI.get(model, attr, ci) ≈ start
+                    returned_start = try
+                        MOI.get(model, attr, ci)
+                    catch err
+                        # For a Constraint bridge for which the map is not invertible, the constraint primal cannot
+                        # be inverted
+                        _runtests_error_handler(
+                            err,
+                            Bridge <: MOI.Bridges.Constraint.AbstractBridge &&
+                                cannot_unbridge,
+                        )
+                        continue
```

Comment:
See the coverage. By my new rule, you must add a test case that catches these lines.